### PR TITLE
update for vulkano 0.23

### PR DESCRIPTION
**Pull request by ryanbarry**
Hello, I came to your library from egui's readme, but found it requires vulkano v0.22 even though they've moved to v0.23. Then I noticed a fork of your library, and found the author had made the changes necessary to use with vulkano 0.23, tested it for myself and found it works!

Would you accept the patch and update your library? Being able to use the crates.io version would be convenient!

Thanks to both you and @Arc-blroth

### CODE REVIEW

File: examples/example.rs [hunk 1]
```diff
@@ -431,16 +420,17 @@ impl Benchmark {
             .enumerate()
             .map(|(i, v)| Value::new(i as f64, *v * 1000.0));
         let curve = Curve::from_values_iter(iter).color(Color32::BLUE);
-        let red = Stroke::new(2.0, Color32::RED);
-        let zero = HLine::new(0.0, Stroke::none());
-        let target = HLine::new(1000.0 / 60.0, red);
+        // TODO: Hlines are now private and I'm not sure why
```
Comment (derivator) [inline examples/example.rs:423]: [Fixed in egui 0.13](https://github.com/emilk/egui/pull/422).

**Comment by Arc-blroth** [comment]
egui 0.13.0 added back HLines, though for some reason there is no setter on HLine for the stroke (I opened an issue about that, see emilk/egui#524). Because of that, in the above commit the example UI displays a light blue line for the target FPS rather than the intended red line.

**Comment by derivator** [comment]
Thanks for your work! I want to merge this. However, to keep git history clean, I would prefer if you could [rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) on my main branch and squash. I would especially like to avoid spaghetti merge commits like 41c21bd.

**Comment by Arc-blroth** [comment]
Rebased and squashed 👍

**Pull request merged**